bgp/gobgp/plugin.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2017 Pantheon technologies s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//Package gobgp contains Ligato GoBGP BGP Plugin implementation
+package gobgp
+
+import (
+	"fmt"
+	"github.com/ligato/bgp-agent/bgp"
+	log "github.com/ligato/cn-infra/logging/logrus"
```

Comment:
let's not be dependent on implementation, use interface (best would be to use go interface PluginConfig, since it allows to create child loggers that might be usefull for original gobgp)

Reply:
For docs:"PluginConfig is API for plugins to access configuration." 
How should PluginConfig help in this situation?

Reply:
Using logging.Logger interface from cn-infra/logging...

Reply:
Using logging.Logger interface from cn-infra/logging...